packages/esm-service-queues-app/src/active-visits/active-visits-table.resource.ts

```diff
@@ -236,23 +256,15 @@ export function useVisitQueueEntries(currServiceName: string, locationUuid: stri
     queueComingFrom: visitQueueEntry.queueEntry?.queueComingFrom?.name,
   });
 
-  let mappedVisitQueueEntries;
-
-  if (!currServiceName || currServiceName == t('all', 'All')) {
-    mappedVisitQueueEntries = data?.data?.results?.map(mapVisitQueueEntryProperties);
-  } else {
-    mappedVisitQueueEntries = data?.data?.results
-      ?.map(mapVisitQueueEntryProperties)
-      .filter((data) => data.service == currServiceName);
+  const mappedVisitQueueEntries = visitQueueEntries?.map(useVisitQueueEntryProperties);
+  if (currServiceName && currServiceName !== t('all', 'All')) {
```

Comment:
On the other hand, it looks like we _need_ the translation hook here?

Reply:
Good catch. Thanks. Missed this when I was refactoring the function. I'm surprised this didn't given me any errors.

Reply:
seems odd that we are basing logic on string comparisions to a localized string, but I suspect this is the existing logic, not something you are introducing?

Reply:
correct. This is old code.